FLT/AutomorphicRepresentation/Example.lean

```diff
@@ -338,6 +357,153 @@ noncomputable abbrev i₁ : ℚ →ₐ[ℤ] QHat := Algebra.TensorProduct.includ
 lemma injective_rat :
     Function.Injective i₁ := RingHom.injective i₁.toRingHom
 
+theorem PNat.lcm_comm (m n : ℕ+) : PNat.lcm m n = PNat.lcm n m := PNat.eq <| by
+  simp [Nat.lcm_comm]
+
+lemma lowestTerms (x : QHat) : (∃ N z, IsCoprime N z ∧ x = (1 / N : ℚ) ⊗ₜ z) ∧
+    (∀ N₁ N₂ z₁ z₂,
+    IsCoprime N₁ z₁ ∧ IsCoprime N₂ z₂ ∧ (1 / N₁ : ℚ) ⊗ₜ z₁ = (1 / N₂ : ℚ) ⊗ₜ[ℤ] z₂ →
+      N₁ = N₂ ∧ z₁ = z₂) := by
```

Comment:
This is surely two different lemmas? The second clause doesn't even mention x. I guess it could be an `∃!` but in practice is this even more difficult to use?